backend/benefit/applications/services/pdf_templates/benefit_with_de_minimis_aid.html

```diff
@@ -14,16 +14,32 @@
             <th class="text-right" scope="col">Yhteensä</th>
         </tr>
         {% for app in apps %}
+            {% if app.ahjo_rows.count() < 2 %}
             <tr>
                 <td>{{ app.ahjo_application_number }}</td>
                 <td>{{ app.company_name }}</td>
                 <td>{{ app.company.business_id }}</td>
                 <td>{{ app.employee.first_name }} {{ app.employee.last_name }}</td>
                 <td>{{ app.start_date.strftime("%d.%m.%Y") }} - {{ app.end_date.strftime("%d.%m.%Y")}}</td>
                 <td>{{ app.get_benefit_type_display() }}</td>
-                <td class="text-right">{{ "%.0f €" | format(app.benefit_amount/(app.duration_in_months | round | int))}} </td>
+                <td class="text-right">{{ "%.0f €" | format(app.benefit_amount/(app.duration_in_months | round(2)))}} </td>
```

Comment:
Should we actually use app.calculation.duration_in_months, as that the handlers might grant the benefit for a shorter time range than the applicant asked for?

Reply:
Yes, nice catch. `app.duration_in_months` was used because previously we didn't have a separated calculation model. Fixed now